.all-contributorsrc

```diff
@@ -5,6 +5,7 @@
   "repoType": "github",
   "commit": false,
   "contributorsPerLine": 6,
+  "linkToUsage": true,
```

Comment:
Should we document this new option in the README?

Reply:
@gr2m all done https://github.com/all-contributors/all-contributors/pull/646 🚀

Reply:
@gr2m anything else we missed?